frontend/src/api.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios';
+
+const API_URL = 'http://localhost:3003';
+
+export async function getRooms() {
+  try {
+    return axios.get(`${API_URL}/rooms`);
+  } catch (err) {
+    return [];
```

Comment:
when the result of request is not successful you need to inform user about it, return empty array is not good